File: godfather/roles/town/vanilla.py
from godfather.roles.base import Role
from godfather.roles.mixins import Townie

DESCRIPTION = 'You have no powers.'


class Vanilla(Townie, Role):
    """
    A regular townie without any powers.

    - Win Condition: Lynch every criminal and evildoer
    """
    name = 'Vanilla'
    description = DESCRIPTION

File: src/lib/mafia/roles/town/Vanilla.ts
import Townie from '@mafia/mixins/Townie';
import Role from '@mafia/structures/Role';
import type Player from '@mafia/structures/Player';

class Vanilla extends Role {
	public name = 'Vanilla';

	public constructor(player: Player) {
		super(player);
		this.description = this.game.t('roles/town:vanillaDescription');
	}
}

export default Townie(Vanilla);
